### PR TITLE
Add UserAgent

### PR DESCRIPTION
**Pull request by borisyankov**
* Polish scroll behavior of MessageList (when there is one or very few messages, align to bottom; do not 'bounce' scroll when back)
* Add react-native-device-info
* Send userAgent string with api requests

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@borisyankov, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by neerajwahi** [comment]
@borisyankov what's the rationale for turning bouncing off? That's generally expected behavior when scrolling on iOS I think

**Comment by borisyankov** [comment]
No other chat app has bouncing on the message list.

File: package.json [hunk 1]
```diff
@@ -29,10 +29,11 @@
     "react": "^15.4.1",
     "react-addons-shallow-compare": "^15.4.1",
     "react-native": "^0.39.2",
+    "react-native-device-info": "^0.9.7",
     "react-native-drawer": "^2.3.0",
     "react-native-scrollable-tab-view": "^0.7.0",
     "react-native-vector-icons": "^3.0.0",
-    "react-redux": "^4.4.6",
+    "react-redux": "^5.0.1",
```
Comment (neerajwahi) [inline package.json]: Please do version updates in separate PRs

File: src/message-list/InfiniteScrollView.js [hunk 2]
```diff
@@ -70,6 +72,8 @@ class InfiniteScrollView extends React.Component {
         scrollEventThrottle={DEFAULT_SCROLL_CALLBACK_THROTTLE}
         stickyHeaderIndices={this.props.stickyHeaderIndices}
         anchorMode
+        bounces={false}
+        scrollEventThrottle={1000}
```
Comment (neerajwahi) [inline src/message-list/InfiniteScrollView.js]: Please remove this. It's a duplicate of the same key above

File: src/message-list/MessageList.js [hunk 3]
```diff
@@ -24,11 +27,13 @@ export default class MessageList extends React.PureComponent {
     return (
       <InfiniteScrollView
         style={styles.list}
+        contentContainerStyle={styles.container}
         automaticallyAdjustContentInset="false"
         autoScrollToBottom={caughtUp}
         stickyHeaderIndices={headerIndices}
         onStartReached={fetchOlder}
         onEndReached={fetchNewer}
+        onScroll={e => {}}
```
Comment (neerajwahi) [inline src/message-list/MessageList.js]: Why is this needed?

Comment (borisyankov) [inline src/message-list/MessageList.js]: Because not yet implemented, but not having one throws, so temporarily an empty function.
